cime_config/testdefs/testmods_dirs/clm/def_cmcc/user_nl_datm_streams

```diff
@@ -0,0 +1,44 @@
+presaero.clim_1850:datafiles=/work/csp/dp16116/inputdata/atm/cam/chem/trop_mozart_aero/aero/aerosoldep_WACCM.ensmean_monthly_hist_1849-2015_0.9x1.25_CMIP6_c180926.nc
```

Comment:
If all of these new settings are introduced - why not add a new datm stream. I'm happy to talk about this.

Reply:
It would be very useful. I look forward to talking about this with you.